infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -122,13 +126,21 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
     event ClaimableChanged(address indexed admin, bool newClaimableState);
     event WhitelistAmountUpdatedByAdmin(address indexed redeemer, uint16 newAmount);
     event WhitelistAmountRedeemed(address indexed redeemer, uint16 newAmount);
+    event RewardClaimedUSDC(address indexed claimer, uint256 usdcAmount);
 
     /**
      * @notice Initializes the NodeLicense contract.
      * 
      */
 
-    function initialize(address _xaiAddress,  address _esXaiAddress, address ethPriceFeedAddress, address xaiPriceFeedAddress, address airdropAdmin) public reinitializer(3) {
+    function initialize(
+        address _xaiAddress,  
+        address _esXaiAddress, 
+        address ethPriceFeedAddress, 
+        address xaiPriceFeedAddress, 
+        address airdropAdmin,
+        address _usdcAddress
```

Comment:
Not sure if it should be in this ticket or have a new ticket, but we'll also need to update the deployment script for this upgrade adding the new param.

Reply:
Good point, @craigbranscom please update the one line in `infrastructure\smart-contracts\scripts\tiny-keys\deployTinyKeys.mjs:67`

Reply:
Updated the deploy script